metadata.json

```diff
@@ -7,58 +7,35 @@
   "source": "https://github.com/voxpupuli/puppet-letsencrypt",
   "project_page": "https://github.com/voxpupuli/puppet-letsencrypt",
   "issues_url": "https://github.com/voxpupuli/puppet-letsencrypt/issues",
-  "tags": [
-    "letsencrypt",
-    "let's encrypt",
-    "certbot",
-    "acme"
-  ],
+  "tags": ["letsencrypt", "let's encrypt", "certbot", "acme"],
```

Comment:
Would you mind keeping the previous formatting? The multiple lines make it easier to add/remove entries and keep the diffs smaller.

Reply:
Sure thing! I think vim+coc auto 'fixed' this for me.